examples/tools/scene_viewer/morph_viewer_plugin.rs

```diff
@@ -43,34 +43,34 @@ const AVAILABLE_KEYS: [MorphKey; 56] = [
     MorphKey::new("7", &[], KeyCode::Digit7),
     MorphKey::new("8", &[], KeyCode::Digit8),
     MorphKey::new("9", &[], KeyCode::Digit9),
-    MorphKey::new("lshift-R", &[KeyCode::ShiftLeft], KeyCode::KeyR),
-    MorphKey::new("lshift-T", &[KeyCode::ShiftLeft], KeyCode::KeyT),
-    MorphKey::new("lshift-Z", &[KeyCode::ShiftLeft], KeyCode::KeyZ),
-    MorphKey::new("lshift-I", &[KeyCode::ShiftLeft], KeyCode::KeyI),
-    MorphKey::new("lshift-O", &[KeyCode::ShiftLeft], KeyCode::KeyO),
-    MorphKey::new("lshift-P", &[KeyCode::ShiftLeft], KeyCode::KeyP),
-    MorphKey::new("lshift-F", &[KeyCode::ShiftLeft], KeyCode::KeyF),
-    MorphKey::new("lshift-G", &[KeyCode::ShiftLeft], KeyCode::KeyG),
-    MorphKey::new("lshift-H", &[KeyCode::ShiftLeft], KeyCode::KeyH),
-    MorphKey::new("lshift-J", &[KeyCode::ShiftLeft], KeyCode::KeyJ),
-    MorphKey::new("lshift-K", &[KeyCode::ShiftLeft], KeyCode::KeyK),
-    MorphKey::new("lshift-Y", &[KeyCode::ShiftLeft], KeyCode::KeyY),
-    MorphKey::new("lshift-X", &[KeyCode::ShiftLeft], KeyCode::KeyX),
-    MorphKey::new("lshift-C", &[KeyCode::ShiftLeft], KeyCode::KeyC),
-    MorphKey::new("lshift-V", &[KeyCode::ShiftLeft], KeyCode::KeyV),
-    MorphKey::new("lshift-B", &[KeyCode::ShiftLeft], KeyCode::KeyB),
-    MorphKey::new("lshift-N", &[KeyCode::ShiftLeft], KeyCode::KeyN),
-    MorphKey::new("lshift-M", &[KeyCode::ShiftLeft], KeyCode::KeyM),
-    MorphKey::new("lshift-0", &[KeyCode::ShiftLeft], KeyCode::Digit0),
-    MorphKey::new("lshift-1", &[KeyCode::ShiftLeft], KeyCode::Digit1),
-    MorphKey::new("lshift-2", &[KeyCode::ShiftLeft], KeyCode::Digit2),
-    MorphKey::new("lshift-3", &[KeyCode::ShiftLeft], KeyCode::Digit3),
-    MorphKey::new("lshift-4", &[KeyCode::ShiftLeft], KeyCode::Digit4),
-    MorphKey::new("lshift-5", &[KeyCode::ShiftLeft], KeyCode::Digit5),
-    MorphKey::new("lshift-6", &[KeyCode::ShiftLeft], KeyCode::Digit6),
-    MorphKey::new("lshift-7", &[KeyCode::ShiftLeft], KeyCode::Digit7),
-    MorphKey::new("lshift-8", &[KeyCode::ShiftLeft], KeyCode::Digit8),
-    MorphKey::new("lshift-9", &[KeyCode::ShiftLeft], KeyCode::Digit9),
```

Comment:
why? this probably breaks things

Reply:
"lshift" isn't in the dictionary. Should we add it to the dictionary? I'll run the example to see if it still works.